### PR TITLE
Add python client documentation 

### PR DESCRIPTION
**Pull request by guy-har**
This PR:
1. Changes the docker image used to create the python client to support Jekyll which was done on [this](https://github.com/treeverse/openapi-generator/tree/v5.1.0.1) forked version of openapi-generator
2. Adds an action that commits and pushes the new version to https://github.com/treeverse/docs-lakefs-python
3. Adds a reference to the python documentation in our lakeFS documentation

### CODE REVIEW

**Comment by guy-har** [comment]
closes #2136

File: clients/python/Gemfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+source 'https://rubygems.org'
+gem 'github-pages', group: :jekyll_plugins
```
Comment (johnnyaug) [inline clients/python/Gemfile:2]: Please add Gemfile, Gemfile.lock  and _config.yml to `.gitattributes` with `linguist-generated=false`.
This way they will appear normally in the diff.
While you're at it, also  lakefs_client/client.py :)

**Review by johnnyaug — CHANGES_REQUESTED** [review]

File: docs/integrations/python.md [hunk 2]
```diff
@@ -169,6 +169,11 @@ client.refs.diff_refs(repository='example-repo', left_ref='experiment-aggregatio
 # []
 ```
 
+## Python client documentation
+
+For the documentation of lakeFS’s python package, see [https://pydocs.lakefs.io](https://pydocs.lakefs.io)
```
[the remark this answers is not in the document]
Comment (guy-har) [reply docs/integrations/python.md:174]: I Considered that but Given that the python docs are automatically generated, I prefer that readers will go through our documentation before. WDYT?

File: .github/workflows/pypi.yaml [hunk 3]
```diff
@@ -7,29 +7,63 @@ on:
...
+        ruby-version: 2.6
+        bundler-cache: true # runs 'bundle install' and caches installed gems automatically
+
+    - name: Build release as latest
+      working-directory: clients/python
+      run: bundle exec jekyll build -d _site -b /
+
+    - name: Build release ${{ steps.docver.outputs.tag }}
+      working-directory: clients/python
+      run: bundle exec jekyll build -d _site/${{ steps.docver.outputs.tag }} -b /${{ steps.docver.outputs.tag }}
```
[the remark this answers is not in the document]
Comment (guy-har) [reply .github/workflows/pypi.yaml:58]: Nice :) 
Added

File: clients/.gitattributes [hunk 4]
```diff
@@ -1,2 +1,5 @@
 java/**		linguist-generated
 python/**	linguist-generated
+python/Gemfile linguist-generated=false
+python/Gemfile.lock linguist-generated=false
+python/_config.yml linguist-generated=false
```
Comment (johnnyaug) [inline clients/.gitattributes:5]: add newline

**Review by johnnyaug — APPROVED** [review]
Nice!!

**Pull request merged**